File: Trip-Planner/src/app/classes/Activiteit.ts
import { Plaats } from "./Plaats";
import { Trip } from "./Trip";

export class Activiteit {
    activiteitId: number;
    activiteitNaam:string;
    plaats: Plaats;
    startDatum: Date;
    eindDatum: Date;
    personenAantal: number;
    trip: Trip;
    
    constructor(activiteitId: number, activiteitNaam:string, plaats: Plaats, startDatum: Date, eindDatum: Date, personenAantal: number, trip: Trip) {
        this.activiteitId = activiteitId;
        this.activiteitNaam =activiteitNaam;
        this.plaats = plaats;
        this.startDatum = startDatum;
        this.eindDatum = eindDatum;
        this.personenAantal = personenAantal;
        this.trip = trip;
    }

}

File: TripPlannerAPI/TripPlannerDAL/Entity/Activiteit.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TripPlannerDAL.Entity
{
    public class Activiteit
    {
        [Key]
        public int activiteitId { get; set; }
        public string activiteitNaam { get; set; }
        public int? plaatsId { get; set; }
        public Plaats? plaats { get; set; }
        public DateTime startDatum { get; set; }
        public DateTime eindDatum { get; set; }
        public int personenAantal { get; set; }
        public int? tripId { get; set; }
        public Trip? trip { get; set; }


    }
}
